test/e2e/e2e_test.go

```diff
@@ -73,6 +75,11 @@ func init() {
 	})
 }
 
+func TestMain(m *testing.M) {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Why not just move this into `init()`? How is TestMain used?

Reply:
Init is executed for each package when it is imported. That means if several packages have RNG seeding in their inits, all of them will be executed. This is not necessary.
Also, init methods with side effects that span packages doesn't feel right.
See https://golang.org/pkg/testing/#hdr-Main

Reply:
I think `rand.Seed()` should only ever be called in `main()` and in `TestMain()` i.e. at the program's entry point.